<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SafeChat - Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>chat.png</normaloff>chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server IP :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serveurIP">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server Port  :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="serveurPort">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>50885</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonConnexion">
       <property name="text">
        <string>Connexion</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="pixmap">
          <pixmap>chat.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>SafeChat</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Pseudo :</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pseudo">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>_my_pseudo_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSendPseudo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Send Pseudo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="listeMessages">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Choose a pseudo a click on the Send button</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Message :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="message">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>_clear_message_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Send Message</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>chat.png</normaloff>chat.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Private</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Create connexion with : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePseudo">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>_pseudo_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbTryConnection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Try Connection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Send a message To : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbClientsSecured"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="tePrivate">
         <property name="placeholderText">
          <string>Enter the pseudo of the person you want to discuss with , and click on Try Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="leSafeMsg">
           <property name="placeholderText">
            <string>_crypted_message_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSendSecured">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Send Message</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Keys</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="tePubKey"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChangePubKey">
             <property name="text">
              <string>Change Public Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTextEdit" name="tePrivKey"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbChangePrivKey">
             <property name="text">
              <string>Change Private Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
